samples/mini-games/src/godotMain/kotlin/example/TestingClass.kt

```diff
@@ -1,6 +1,7 @@
 package example
 
 import godot.annotation.*
+import godot.core.GodotResource
```

Comment:
Resource is generated and so is in godot package.

Reply:
ok i move the dummyClass